app/views/games/summary.html.haml

```diff
@@ -10,4 +10,6 @@
 - if @flags_per_hour.count > 1
   %h3.muted
     Flag Submissions
-    = line_chart @line_chart_data, colors: ['#b8d12f', '#00abca']
+    = line_chart(@line_chart_data, library: {chart: {zoomType: 'x'}}, colors: ['#b8d12f', '#00abca'])
+  %p.muted
+    = t('game.zoom_help')
```

Comment:
There is a setting in your editor to automatically add new lines to the end of a file, generally the best practice is to have that enabled: https://thoughtbot.com/blog/no-newline-at-end-of-file

Reply:
Got it, and enabled now, thanks.

Reply:
Can you amend this commit so the change is applied to this file as well?

Reply:
Done